.openshift/action_hooks/build

```diff
@@ -0,0 +1 @@
+export OPENSHIFT_PYTHON_WSGI_APPLICATION=memepol/wsgi.py
```

Comment:
That's probably why it wasn't working and I still had to pass it on the command line to create the app, thanksssssssssssss